var/spack/repos/builtin/packages/julia/package.py

```diff
@@ -31,13 +31,15 @@
 class Julia(Package):
     """The Julia Language: A fresh approach to technical computing"""
     homepage = "http://julialang.org"
-    url      = "https://github.com/JuliaLang/julia/releases/download/v0.4.3/julia-0.4.3-full.tar.gz"
```

Comment:
Please switch this line back. It is exempt from flake8 line length checks (see [here](https://github.com/LLNL/spack/blob/develop/lib/spack/spack/cmd/flake8.py#L55)).